src/main/java/seedu/address/model/person/Email.java

```diff
@@ -44,29 +43,36 @@
      */
     public Email(String email) {
         requireNonNull(email);
-        checkArgument(isValidEmail(email), MESSAGE_CONSTRAINTS);
+        checkArgument(isValidEmailAddress(email), MESSAGE_CONSTRAINTS);
         value = email;
     }
 
     /**
-     * Returns if a given string is a valid email.
+     * Private constructor for Empty Email.
      */
-    public static boolean isValidEmail(String test) {
-        return isEmptyEmail(test) || test.matches(VALIDATION_REGEX);
+    private Email() {
+        value = "";
     }
 
     /**
-     * Returns true if the given string is the default empty email.
+     * Returns true if a given string is a valid email.
      */
-    public static boolean isEmptyEmail(String test) {
-        return test.equals(EMPTY_EMAIL);
+    public static boolean isValidEmailAddress(String test) {
```

Comment:
Perhaps a variable name that illustrates the difference between isValidEmailAddress and isValidEmail more intuitively?